index.bs

```diff
@@ -3977,6 +3991,9 @@ a numbered step. If outdented, it (today) is rendered as a bullet in the midst o
 
             - |authenticatorData|
             - |signature|
+            - |transport|
+
+                    Note: the returned transport value may be `null`.
```

Comment:
The authenticator does not return to the UA (i.e. the browser) the transport. The UA knows from the mechanism it used to communicate with the authenticator. Remove this.

```suggestion
```